Vault 43914/ldap rotate onread feature docs (#2955)
hashicorp/web-unified-docs: content/vault/v2.x/content/docs/secrets/ldap/index.mdx

  configure an administrator level LDAP bind account on the plugin that has
  permission to change other LDAP user passwords.

### Rotate on read ((#rotate-on-read))

@include '../../../global/partials/alerts/ent/ent-only.mdx'

Vault can rotate a static role credential each time it is read from
`/{mount_path}/static-cred/{role_name}`, subject to a configurable cooldown
period that prevents redundant rotations when multiple clients read the same
role in quick succession. Rotate-on-read is independent of any

> schedule-based rotation on the role.

Reviewer: ```suggestion
Vault can rotate a static role credential each time it reads the credential from
`/{mount_path}/static-cred/{role_name}` with a configurable cooldown period that
prevents redundant rotations when multiple clients read the same
role in quick succession. Vault manages rotate-on-read settings independently from
any schedule-based rotation on the role.
```

- `rotate_on_read` `(bool: false)` - Rotate credentials on read when `true`.

>   Configurable at the mount level or overridden per role.

Reviewer: ```suggestion
  Configurable at the mount or role level.
```

- `rotate_on_read_cooldown` `(duration: "1m")` - Minimum time between

>   rotations. Setting to `0` disables the cooldown and rotates on every read.

Reviewer: ```suggestion
  rotations. Set the duration to `0` to disable the cooldown and rotate on every read.
```
